### PR TITLE
Feature/my_hadoop_v0.40a_sdsc

### PR DESCRIPTION
**Pull request by molden**

### CODE REVIEW

**Comment by boegel** [comment]
@ehiggs: please review?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3583b83ccfa2a28df65d for a full test report.

**Comment by boegel** [comment]
Works, but needs a style review...

@wpoely86?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0344ce563494b43a96ab for a full test report.

**Comment by boegel** [comment]
@wpoely86 ping on style review?

File: easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'myHadoop'
+version = 'v0.40a-sdsc'
+
+homepage = 'https://github.com/glennklockwood/myhadoop'
+description = """myhadoop provides a framework for launching Hadoop clusters 
+ within traditional high-performance compute clusters and supercomputers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb]: Put the `version` to an empty string or else the deps are not loaded

File: easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'myHadoop'
+version = 'v0.40a-sdsc'
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb]: is the `-sdsc` part of the version or is it a versionsuffix?

File: easybuild/easyconfigs/m/myHadoop/myhadoop-configure.patch [hunk 3]
```diff
@@ -0,0 +1,72 @@
+diff -ru a/bin/myhadoop-configure.sh b/bin/myhadoop-configure.sh
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/myHadoop/myhadoop-configure.patch]: Please add a top comment about what this patch does and who created it.

File: easybuild/easyconfigs/m/myHadoop/torque.qsub.patch [hunk 4]
```diff
@@ -0,0 +1,56 @@
+diff -ru a/examples/torque.qsub b/examples/torque.qsub
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/myHadoop/torque.qsub.patch]: same here

File: easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb (hunk 2 above)
Comment (molden) [inline easybuild/easyconfigs/m/myHadoop/myHadoop-v0.40a-sdsc.eb]: Is part of version

**Comment by migueldiascosta** [comment]
@molden, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**